src/utils/string.utils.ts

```diff
@@ -0,0 +1,16 @@
+const MAX_WORDS_PER_TITLE_LINE = 5;
+
+export const splitTextHeader = (text = '') => {
+  let textLine1 = text;
```

Comment:
Why text can be an empty string?
And if it can be empty, maybe return it immediately?

Reply:
I put empty string here like default parameter, in this case I not need to check for undefined.
Text can come to component like undefined, in this case this function will not evaluate